### PR TITLE
edbrowse: 3.7.4 -> 3.7.6

### PR DESCRIPTION
**Pull request by equirosa**
###### Motivation for this change

Newest version has a bunch of bug fixes,  latest version within nixpkgs is from over a year ago.
https://github.com/CMB/edbrowse/releases/tag/v3.7.6
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

**Review by jonringer — APPROVED** [review]
diff LGTM
executable still shows usage

```
[2 built, 4 copied (7.0 MiB), 1.1 MiB DL]
https://github.com/NixOS/nixpkgs/pull/75087
1 package were built:
edbrowse
```

**Comment by jonringer** [comment]
@GrahamcOfBorg build edbrowse

**Review by c0bw3b — CHANGES_REQUESTED** [review]
The commit fetched as a patch is now part of the version so it isn't needed anymore

**Comment by equirosa** [comment]
> The commit fetched as a patched is now part of the version so it isn't needed anymore

Ok so this means that it isn't necessary to update it? (A bit confused here :sweat_smile: )

**Comment by c0bw3b** [comment]
No, I just meant that you could remove the following block:

```nix
  patches = [
    # Fix build against recent libcurl
    (fetchpatch {
      url = https://github.com/CMB/edbrowse/commit/5d2b9e21fdf019f461ebe62738d615428d5db963.diff;
      sha256 = "167q8n0syj3iv6lxrbpv4kvb63j4byj4qxrxayy08bah3pss3gky";
    })
  ];
```

because that patch isn't needed with the newer 3.7.6 version.

(also remove `fetchpatch` from the inputs at the top)

**Comment by equirosa** [comment]
> No, I just meant that you could remove the following block:
> 
> ```nix
>   patches = [
>     # Fix build against recent libcurl
>     (fetchpatch {
>       url = https://github.com/CMB/edbrowse/commit/5d2b9e21fdf019f461ebe62738d615428d5db963.diff;
>       sha256 = "167q8n0syj3iv6lxrbpv4kvb63j4byj4qxrxayy08bah3pss3gky";
>     })
>   ];
> ```
> 
> because that patch isn't needed with the newer 3.7.6 version.
> 
> (also remove `fetchpatch` from the inputs at the top)

Ohhhhhh... Got it! Will ammend ASAP :smile:

File: pkgs/applications/editors/edbrowse/default.nix [hunk 1]
```diff
@@ -1,26 +1,11 @@
...
 
-  patches = [
-    # Fix build against recent libcurl
-    (fetchpatch {
-      url = https://github.com/CMB/edbrowse/commit/5d2b9e21fdf019f461ebe62738d615428d5db963.diff;
-      sha256 = "167q8n0syj3iv6lxrbpv4kvb63j4byj4qxrxayy08bah3pss3gky";
-    })
-  ];
-
-  postPatch = ''
```
Comment (c0bw3b) [inline pkgs/applications/editors/edbrowse/default.nix]: The patch was not needed anymore, but on the other hand this `postPatch` shell hook is still needed for the build to succeed.

Even if no patches are explicitly given, the postPatch hook runs as part of [the patch phase](https://nixos.org/nixpkgs/manual/#ssec-patch-phase) of a standard nix build.

Comment (equirosa) [reply pkgs/applications/editors/edbrowse/default.nix]: Ohhh I see... Fixed :sweat_smile:

**Comment by c0bw3b** [comment]
@equirosa just added a commit to update the hash of the source archive.
Without it, we were still building the v3.7.4 with the sources available from the cache and matching the give SHA256 hash.

**Review by c0bw3b — APPROVED** [review]
Result of `nix-review pr 75087`
<details>
  <summary>1 package were build:</summary>

  - edbrowse
</details>

---

```console
$ ./results/edbrowse/bin/edbrowse -v
3.7.6
```

**Pull request merged**